app/src/lib/stores/app-store.ts

```diff
@@ -1483,14 +1483,13 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     const repository = selection.repository
     const repoState = this.repositoryStateCache.get(repository)
-    // check if conflicts exist by inspecting conflictState: null means no conflicts exist
-    const conflictsExist = repoState.conflictState !== null
+    const { conflictState } = repoState.changesState
 
-    if (conflictsExist) {
```

Comment:
:art: was the intent here to revert the variable name?

Reply:
Yea, I changed it because TS wouldn't correctly infer the type if I used boolean, so every line after 1490 complained that `conflictState` could be null.  I added a comment to aide in understanding though.